cmd/otel-allocator/server/server_test.go

```diff
@@ -15,15 +15,16 @@
 package server
 
 import (
-	"crypto/rand"
 	"encoding/json"
 	"fmt"
 	"io"
-	"math/big"
+	"math/rand"
```

Comment:
I thought we had a linter that discourages the use of `math/rand` in favor of `crypto/rand`, i guess it's fine given this is just a test.

Reply:
oh now i see, you're disabling the linter down there, that's fine